src/StackExchange.Redis/StringSplits.cs

```diff
@@ -6,4 +6,9 @@ internal static class StringSplits
             Space = { ' ' },
             Comma = { ',' };
     }
+
+    internal static class Messages
+    {
+        public const string PreferReplica = "Starting with Redis version 5, redis has moved to 'replica' terminology.";
```

Comment:
I think this is fine for most places, but probably a better "use this property" instead directly pointing to the replacement would be better - I'm happy to do that commit if cool with it. Related: a "Messages" internal class name here probably isn't awesome, should be `ObsoleteMessages` or something since Message is a core thing.

Cool with me putting specific messages and removing this, bypassing the name issue?

Reply:
How about as written plus suffix " please use " + nameof

Reply:
Agree that sounds good - will adjust